lib/active_remote/persistence.rb

```diff
@@ -245,9 +245,9 @@ def create
     # are created, existing records are updated. If the record is marked as
     # readonly, an ActiveRemote::ReadOnlyRemoteRecord is raised.
     #
-    def create_or_update
+    def create_or_update(*args)
       raise ReadOnlyRemoteRecord if readonly?
-      new_record? ? create : update
+      new_record? ? create : update(*args)
```

Comment:
Weird. I'm guessing they're doing that so the method arity doesn't change when [dirty tracking](https://github.com/liveh2o/active_remote/blob/master/lib/active_remote/dirty.rb#L87) is included.